integration_tests/fixtures/applicationData.json

```diff
@@ -154,9 +154,10 @@
       "isEngagedWithCommunity": "no",
       "servicesDetail": "",
       "isEngagedWithServicesInCustody": "no",
-      "hasPrescribedMedication": "no",
-      "medicationDetail": "",
-      "medicationIssues": ""
+      "canManageMedication": "notPrescribedMedication",
+      "canManageMedicationNotes": "",
```

Comment:
minor thing, but I think if the user answers `notPrescribedMedication` then there won't be any keys in the application data for the next three fields.

Reply:
Good spot. It looks like we include empty keys in other places too, but in any event, it shouldn't give us a false-positive, so I think this is OK to go in as is.